### DISCUSSION TITLE
Suggestion for imrpovements

### DISCUSSION THREAD

**Discussion by Xteri08** [Robby3St/TradePlugin#52] [Ideas]
Hello sir hope you are doing great and this contribution towards communtiy is always gratefull and respectiable.
I would first like to suggest somethings:

Add economy trade so trade can be done with ingame currency too.
also please with everyupdate make sure to be dupe glich free as every trade plugin get discovered onto it.
Please make sure to add geyser support so the gui is userfriendly for geyser players too.

**Comment by Robby3St** [comment]
Hello! Sorry for my long hesitation for answering you. I'm happy you've found my plugin. Unfortunately, I'm just busy at the moment with my exams at university. But can you give me a hint in which way you would like to see trading with ingame economy? Do you like to have an optional ingame economy by the trade plugin itself or by using an external plugin? Can you recommend an external economy plugin, so I can implement the usage in my plugin for it? 

Moreover, can you tell me if the plugin is already geyser compatible? I didn't hear of geyser before, so I dont't exactly know, where the compatibility might fail.

Best regards
